### PR TITLE
feat: unread messages badge

### PR DESCRIPTION
**Pull request by LomyW**
**Explanation**
A badge on the navigation should show, if a user has unread messages on the chat page.

**What was done?**
- [x] added badge on the navigation bar to display if a unread message exists
- [x] added also a count, which show how many chats with unread messages exist
- [x] extended `ChatContext` to get informations about unread messages
- [x] added placeholder

### CODE REVIEW

File: src/context/ChatContext.tsx [hunk 1]
```diff
@@ -65,12 +72,25 @@ export const LFChatProvider: React.FC<{ children: ReactNode }> = ({ children })
...
     }, [talkLoaded, loading, sessionState]);
 
-    const contextValue = useMemo(() => ({ session, talkLoaded }), [session, talkLoaded]);
+    useEffect(() => {
+        if (!session) return;
+        const unreads = session.unreads;
+
+        unreads.onChange((message) => {
+            setHasUnreadMessages(true);
+            setUnreadMessagesCount(message.length);
```
Comment (LomyW) [inline src/context/ChatContext.tsx:81]: for now we can't access `unreadMessagesCount`, thats why we have to use the `length`

Comment (timidak) [reply src/context/ChatContext.tsx:81]: Why?

Comment (LomyW) [reply src/context/ChatContext.tsx:81]: only have access to `lastMessage`

- https://talkjs.com/docs/Reference/JavaScript_Chat_SDK/Session/#UnreadConversation
- https://talkjs.com/docs/Reference/JavaScript_Chat_SDK/Session/#Unreads__onChange

File: src/pages/Chat.tsx [hunk 2]
```diff
@@ -65,10 +65,13 @@ const Chat: React.FC = () => {
 
     useEffect(() => {
         if (!session) return;
-        const inbox = session.createInbox({ showChatHeader: !isMobile, showMobileBackButton: false });
+        const inbox = session.createInbox({
+            showChatHeader: !isMobile,
+            showMobileBackButton: false,
+            messageField: { visible: { access: ['==', 'ReadWrite'] }, placeholder: t('chat.placeholder') },
```
Comment (LomyW) [inline src/pages/Chat.tsx:71]: add placeholder on message field in chat again, was lost after merging

**Review by timidak — APPROVED** [review]

**Pull request merged**